Donut/templates/layout.html

```diff
@@ -5,21 +5,21 @@
         
         <link rel="stylesheet" type="text/css" 
               href="{{ url_for('static', filename='css/style.css') }}" />
-        <!-- Google Fonts -->
-        <link href='http://fonts.googleapis.com/css?family=EB+Garamond|Roboto' rel='stylesheet' type='text/css'>
+        {# Google Fonts #}
+        <link href='https://fonts.googleapis.com/css?family=EB+Garamond|Roboto' rel='stylesheet' type='text/css'>
 
-        <!-- Bootstrap core CSS -->
+        {# Bootstrap core CSS #}
         <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
-        <!-- Bootstrap theme -->
+        {# Bootstrap theme #}
 
-        <!-- Just for debugging purposes. Don't actually copy this line! -->
-        <!--[if lt IE 9]><script src="../../docs-assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+        {# Just for debugging purposes. Don't actually copy this line! #}
+        {# [if lt IE 9]><script src="../../docs-assets/js/ie8-responsive-file-warning.js"></script><![endif] #}
 
         <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
-        <!--[if lt IE 9]>
+        {# [if lt IE 9]>
```

Comment:
Uh... I meant using the html comments here, since you're using the conditional comment syntax.